src/plugin/iframe_root/modules/components/signupForm.js

```diff
@@ -432,11 +433,31 @@ define([
             validationFieldBorder: true
         });
 
-        var referralSources = ko.observableArray(referralSourceData);
-        var referralSourceCopy = 'How did you hear about us? (select all that apply)';
+        var referralSourceCopy = 'How did you hear about us?';
```

Comment:
This was my instinct as well, avoided it because I didn't see it elsewhere, but for some reason also used arrow functions later on 😅

Reply:
ES6 is perfectly Ok and encouraged!
Generally these older codebases are incrementally migrated to ES6. I typically convert a file to ES6 when I touch it, but depending on the changes, it may only be actual code within a file that is touched. Typically, var -> const, let; string + string -> interpolated strings; function -> fat arrow; etc.
I seldom find actual issues with old usages (but just issued a narrative viewer fix due to var usage), but I just like a consistent pattern when possible to make code scanning easier, and also some usages are more compact, reduce code stuttering, etc.